### ISSUE TITLE
Could not find the data

### ISSUE THREAD

**Issue by lryffel** [ThisSentenceIsALie/Wirt_Hm#1]
Hi!

I somehow accidentally stumbled upon this repo, and was immediately reminded of my master thesis I wrote under Sebastian Baader in 2020, where I wrote a program to try to prove the meridional rank conjecture for as many knots as possible using surjective homomorphisms to Coxeter groups. Sound familiar? :)

My approach seems to have been slightly different than yours, however, because I did not limit my search to finite Coxeter groups (but also had no guarantee whatsoever that if a surjective homomorphism exists my program spits it out). My program also appears to have been a whole lot slower, as it could only handle up to 11 crossings or so.

I did not immediately find the data you mentioned in the README. It would perhaps be fun to compare our approaches. If you're curious about my thesis, I made my repos public (but they are not very cleaned up, I'm sorry). They are at:

https://github.com/raw-bacon/master-thesis (this one contains a pdf with a table at the end with some human-readable data for up to 10 crossings)
https://github.com/raw-bacon/master-thesis-python (this one contains the source code of a program using the reflection representation to check whether a given Coxeter group is a geometric quotient of a knot group)

**Comment by ThisSentenceIsALie** [comment]
Hi!

Sorry it took me a few days to respond. I'm still in the process of putting together the repository, and I really wasn't expecting anyone to care about it yet!

That's really interesting. We definitely had similar approaches, though we spent most of our time finding ways to guarantee that finding no surjective homomorphism guarantees none exists to a finite Coxeter group. My advisor was Dr. Blair, who I'm pretty sure knows Dr. Baader. Not too surprising, since apparently they have very similar interests.

The data should be along in a few days. We're still pulling it all together. This was all done a couple years ago, and my file organization was... nonoptimal, even back when I remembered where everything was. 

A lot of my code is built off of algorithms developed by one of Dr. Blair's former grad students, Paul Villanueva, and if you're interested in his method for coloring knots and computing the wirtinger number he does a great job explaining his logic on [his github](https://github.com/pommevilla/calc_wirt).
